{"uuid": "b3b50394-b1fb-49a4-95dc-80341a5e2d6e", "children": ["10584145-cbf5-4b0e-bed9-131393a4f891"], "befores": [{"name": "setup_teardown", "status": "passed", "start": 1753550716964, "stop": 1753550716964}], "afters": [{"name": "setup_teardown::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"/Users/ducplm/Documents/auto_test/automation_project/venv/lib/python3.13/site-packages/allure_commons/_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/Users/ducplm/Documents/auto_test/automation_project/venv/lib/python3.13/site-packages/_pytest/fixtures.py\", line 938, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"/Users/ducplm/Documents/auto_test/automation_project/conftest.py\", line 33, in setup_teardown\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1753550716968, "stop": 1753550716968}], "start": 1753550716964, "stop": 1753550716968}